tests/unit/Codeception/StepTest.php

```diff
@@ -114,4 +114,14 @@ public function testSeeMultiLineStringInSingleLine()
         $output = $step->toString(200);
         $this->assertEquals('see "aaaa\nbbbb\nc"', $output);
     }
+
+    public function testFormattedOutput()
+    {
+        $argument = $this->createMock('\Codeception\Step\Argument\FormattedOutput');
```

Comment:
Please use `Codeception\Util\Stub::make` instead. `$this->createMock` is not available in older PHPUnit versions. That's why tests are failing